src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/ReducedTransObjectArray.java

```diff
@@ -0,0 +1,29 @@
+package de.internetsicherheit.brl.bloxberg.cache.ethereum;
+
+import org.web3j.protocol.core.methods.response.Transaction;
+
+import java.util.ArrayList;
+
+public class ReducedTransObjectArray {
+
+    /**
+     * a simple custom Datatype that represents a combination of a blocknumber and the corresponding transactioncount
+          */
+    public ArrayList<String> fromAddress;
```

Comment:
```suggestion
    public final List<String> fromAddress = new ArrayList<>();
```
 If we make fields public, let's at least make the reference immutable. Also, use interface type and diamond operator.

Reply:
ok, so I think I understand everything so far, but what does the empty diamond operator do? generics require a type, right?

Reply:
Thanks to type inference, Java SE 7 and above allow the programmer to substitute an empty pair of angle brackets (<>, called the diamond operator) for a pair of angle brackets containing the one or more type parameters that a sufficiently-close context implies.

So said in simple words: If the type on the left already tells you the generic type, the compiler can infer it and it is not necessary it specify it on the right as well.